src/zope/schema/_field.py

```diff
@@ -751,6 +748,37 @@ def _validate_fields(schema, value):
     return errors
 
 
+class _BoundSchema(object):
```

Comment:
After looking at #57, I think this proxying approach is overkill. It was useful if we were binding to `field.context`, but since we're *not*, just the value being validated, we can just bind each field as we get it in `_validate_fields`, just like `gSVE` does.